teps/0096-pipelines-v1-api.md

```diff
@@ -174,11 +182,110 @@ V1 work should prioritize stabilizing existing beta features that support the [u
+  - See [Handling parameter interpolation in fields not designed for it](https://github.com/tektoncd/pipeline/issues/1530) for more details.
+
+#### Deprecations
+
+- Deprecate `PipelineRun.Timeout` and upgrade `PipelineRun.Timeouts` to beta.
+- Deprecate `Pipeline.Tasks[].Conditions`. Conditions have been replaced by when expressions, which are already in beta.
+- Because PipelineResources are deprecated, remove the `Resources` fields of `Task`, `TaskRun`, `Pipeline`, and `PipelineRun`.
+    - We will proceed with [TEP-0074: Deprecate PipelineResources](https://github.com/tektoncd/community/blob/main/teps/0074-deprecate-pipelineresources.md)
+    independently of Pipelines V1, and merging this TEP as implementable isn't a V1 blocker.
+    However, we should still provide PipelineResource users with a migration path to the V1 API.
+    Additional community feedback is still needed to determine the minimum V1 requirements that will make it usable for PipelineResource users.
+
+#### Behavior flags
+
+Existing feature flags can be found [here](https://github.com/tektoncd/pipeline/blob/main/config/config-feature-flags.yaml).
+The deprecation of the "disable-home-env-overwrite" and "disable-working-directory-overwrite" feature flags has already [been announced](https://github.com/tektoncd/pipeline/blob/main/docs/deprecations.md#deprecation-table). 
+
+| Flag                                          | Current Default | Proposed State                                       |
+| --------------------------------------------- | --------------- | -----------------------------------------------------|
+| disable-affinity-assistant                    | false           | default to true                                      |
+| disable-home-env-overwrite                    | true            | flag removed, home env overwrite feature disabled    |
+| disable-working-directory-overwrite           | true            | flag removed, working dir overwrite feature disabled |
+| disable-creds-init                            | false           | no change                                            |
+| running-in-environment-with-injected-sidecars | true            | no change                                            |
+| require-git-ssh-secret-known-hosts            | false           | no change                                            |
+| enable-tekton-oci-bundles                     | false           | collapsed under enable-api-fields                    |
+| enable-custom-tasks                           | false           | collapsed under enable-api-fields                    |
+| enable-api-fields                             | stable          | default to stable, "beta" option added               |
+| scope-when-expressions-to-task                | false           | feature stabilized and flag removed                  |
+
+We may want to consider standardizing the naming of feature flags (for example, renaming "disable-foo" flags to "enable-foo").
```

Comment:
Usually we may want to use disable and enable to determine if the user has to opt-out or opt-in. I don't really have any problem with having feature flag that start with enable and some with disable

Reply:
fair enough! I'll remove this